sources/packages/backend/package.json

```diff
@@ -42,26 +42,26 @@
     "deploy:camunda:definitions": "ts-node ./workflow/src/deploy.ts"
   },
   "dependencies": {
-    "@aws-sdk/client-s3": "^3.621.0",
+    "@aws-sdk/client-s3": "^3.663.0",
     "@bull-board/api": "^5.15.1",
     "@bull-board/express": "^5.15.1",
```

Comment:
Bull board is also using `path-to-regexp@0.1.7`

![image](https://github.com/user-attachments/assets/53c68b44-8d1c-4608-9820-037ecd3c845e)

Reply:
`path-to-regexp@0.1.7` and `body-parser@1.20.2` are dependencies of `express@4.18.3`. There is no vulnerabilities found using `npm audit` with `@bull-board/express@5.15.1`, so these two dependencies are secured versions. I tried to upgrade the dependencies' versions by installing newer versions for `@bull-board/express`, i.e., `@bull-board/express@5.23.0` and `bull-board/express@6.0.0`. They are using older versions of `path-to-regexp` but updated versions of `body-parser`. 
Screenshot for installing `@bull-board/express@5.23.0`
![image](https://github.com/user-attachments/assets/ca338f68-a41f-4282-bae4-ba948f78b9c2)
![image](https://github.com/user-attachments/assets/e3079ae4-0935-4639-9255-3930c2fb7bad)

Screenshot for installing `@bull-board/express@6.0.0`
![image](https://github.com/user-attachments/assets/ec9abd49-6029-418f-9e00-cce35c8e2f37)
![image](https://github.com/user-attachments/assets/992a6401-9f65-4556-81a4-7e533584f462)

However, `queue-consumer` failed to build with `make queue-consumers` using both `@bull-board/express@5.23.0` and `bull-board/express@6.0.0` unless we make further changes or installing other dependencies.
![image](https://github.com/user-attachments/assets/4169f01b-079a-4c8c-927f-93ae557ec52c)

Reply:
Okay, considering it not a blocker.